packages/gatsby-theme-docs/src/components/top-menu-item.js

```diff
@@ -0,0 +1,185 @@
+import PropTypes from 'prop-types';
+import { navigate } from 'gatsby';
+import { LordIcon, designSystem } from '@commercetools-docs/ui-kit';
+import styled from '@emotion/styled';
+import { css, keyframes } from '@emotion/react';
+import { AngleRightIcon } from '@commercetools-uikit/icons';
+
+const MenuItemWrapper = styled.div`
+  display: flex;
+  padding: ${({ isSmall }) =>
+    isSmall ? '4px 12px 4px 8px' : '8px 12px 8px 8px'};
+  height: 32px;
```

Comment:
Would be nice if the designSystem is being used here instead of defining the pixels here.